src/e2e/puppeteer/__tests__/snapshot.ts

```diff
@@ -302,7 +302,61 @@ describe('Drag and Drop simulation tests', () => {
       - d
     `)
 
-    await dragAndDropThought('a', 'd', { position: 'after' })
+    await dragAndDropThought('a', 'd', { position: 'after' }, true)
+
+    const image = await screenshot()
+    expect(image).toMatchImageSnapshot()
+  })
+
+  it('Check drop hover tests', async () => {
+    await paste(`
+      - a
+      - b
+      - c
+      - d
+    `)
+
+    await dragAndDropThought('a', 'd', { position: 'after' }, false)
+
+    const image = await screenshot()
+    expect(image).toMatchImageSnapshot()
+  })
+
+  it('Check drop target tests, covering DragAndDropThought and DropChild', async () => {
+    await simulateDragAndDrop({ drag: false, drop: true })
+    await paste(`
+      - a
+      - b
+      - c
+      - d
+    `)
+
+    const image = await screenshot()
+    expect(image).toMatchImageSnapshot()
+  })
+
+  it('Check drop target tests, covering DropEnd', async () => {
+    await simulateDragAndDrop({ drag: false, drop: true })
+    await paste(`
+      - a
+        - b
+          - c
+    `)
+
+    const image = await screenshot()
+    expect(image).toMatchImageSnapshot()
+  })
+
+  it('Check drop target tests, covering DropUncle', async () => {
```

Comment:
Please follow the instructions that were given for this test. The cursor needs to be on `c` for the `DropUncle` component to be rendered. Please review the [wiki](https://github.com/cybersemics/em/wiki/Docs#7-drag-and-drop) and check out the [DropUncle component](https://github.com/cybersemics/em/blob/472472069b4637a88c9bbb14f2ce6918ad742b1e/src/components/DropUncle.tsx#L17) if you are unclear about how `DropUncle` is rendered.

> 3\. A test that covers `DropUncle` (with the cursor on `c`)